include/package-pack.mk

```
@@ -359,7 +359,7 @@ else
 			) \
 		), \
 		$$(prov) )" \
-	  $(if $(DEFAULT_VARIANT),--info "provider-priority:100") \
+	  $(if $(DEFAULT_VARIANT),--info "provider-priority:100",--info "provider-priority:1") \
```

(untested)

```suggestion
	  $(if $(DEFAULT_VARIANT),--info "provider-priority:100",$(if $(PROVIDES),--info "provider-priority:1")) \
```

IIUC then this is going to set `provider-priority:1` on all packages and perhaps you want to set it only when its needed?

---

It makes sense, tested and it still works.

---

It looks like that final comma should be gated to the if condition, not come before it. But if it works, it works...

---

Hm, indeed it probably should be gated, though it worked for me just fine this was as well